lua/auto-session/init.lua

```diff
@@ -253,13 +284,13 @@ end
 ---unless a session for the current working directory exists.
 ---@return boolean True if a session exists for the cwd
 function AutoSession.session_exists_for_cwd()
-  local session_file = get_session_file_name(vim.fn.getcwd(-1, -1))
+  local session_file = get_session_file_name(nil)
```

Comment:
passing nil (or nothing) here causes `get_session_file_name` to call `get_session_name` with `use_cwd: true`, preserving the existing behavior